app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.kt

```diff
@@ -157,6 +171,43 @@ class MainActivity : BaseActivity(),
         viewModel.checkPostNotificationsPermission()
     }
 
+    @Composable
+    private fun ScheduleChangesStatistic() {
+        EventFahrplanTheme {
+            scheduleChangeStatisticsUiState.value?.let {
+                ChangesStatisticScreen(
+                    uiState = it,
+                    onConfirm = {
+                        AppRepository.updateScheduleChangesSeen(true)
+                        scheduleChangeStatisticsUiState.value = null
+                        openSessionChanges()
```

Comment:
It feels like this logic should live inside a view model.

Reply:
:heavy_check_mark: Reduced to ... now:

``` kotlin
ChangesStatisticScreen(
    uiState = state,
    onConfirm = { viewModel.onCloseChangesStatisticScreen(shouldOpenSessionChanges = true) },
    onDismiss = { viewModel.onCloseChangesStatisticScreen(shouldOpenSessionChanges = false) },
)
```